### DISCUSSION TITLE
pdf tutorial

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#836] [Help]
Hi all,
2 months ago I read a pdf tutorial about freeplane.
It was a longer article (or series of articles?) taken out of some magazine, I think.
As far as I remember, the color scheme was more in red/white.

Now, I wanted to look it up again, and can't find it anywhere anymore, not even with a general Google search.
Does anyone know where it can be found and where the official link to it was?

Mtia and regards, a.

**Comment by dpolivaev** [comment]
Help->Freeplane handbook
https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane-handbook-fullcircle-parts-1-15.pdf/download

**Reply by autoca** [reply]
ok, thx. that's it.
I looked all over the web (even sourceforge, but did not see it), pressed F1, but looking for other items in the help drop-down was not on my radar.
